projects/is-editable-textbox/src/lib/is-editable-textbox.component.ts

```diff
@@ -1,15 +1,23 @@
-import { ChangeDetectorRef, Component, EventEmitter, forwardRef, Input, OnInit, Output } from '@angular/core';
-import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { ChangeDetectorRef, Component, ElementRef, EventEmitter, forwardRef, Input, OnInit, Output } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, NG_VALIDATORS, AbstractControl } from '@angular/forms';
+
+const IS_EDITABLE_TEXTBOX_VALUE_ACCESSOR: any = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => IsEditableTextboxComponent),
+  multi: true
+};
+
+const IS_EDITABLE_TEXTBOX_VALIDATORS: any = {
+  provide: NG_VALIDATORS,
+  useExisting: forwardRef(() => IsEditableTextboxComponent),
+  multi: true
+}
```

Comment:
please remove validation fucntionality completely. It does not have any sense to have control "self validated" based on input property. Clients should rather provide custom validation via @angular/forms